### PR TITLE
1.12 Terraform documentation

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/DCOS-43624
https://docs.google.com/document/d/1Z25jSs1KZ14XEHeO5823oRHwtt17YJLW6sODtUygC6s/edit
https://www.mindmeister.com/1158553831

## Urgency
- [x] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by anithamuthu79** [comment]
@yishan-lin  and @bernadinm please review and share feedback on this PR.

**Comment by yishan-lin** [comment]
Hey Anitha, we are not going to have a Troubleshooting & FAQ guide in time for 1.12 GA.  :) Please remove for now.

**Comment by anithamuthu79** [comment]
@yishan-lin I've removed "trouble-shooting and faq" folder.

**Comment by anithamuthu79** [comment]
@yishan-lin I've addressed your comments and have asked @niartenyaw to replace 
`You can create a DC/OS cluster on AWS using Terraform` with `You can create a DC/OS cluster on AWS using an Universal Installer` on AWS, Azure, and GCP pages under Mesosphere-supported installation methods on dev PRs.

**Comment by yishan-lin** [comment]
Hey @anithamuthu79 - I don't see the updated content?  Apologies if I'm missing something.

File: pages/1.12/installing/evaluation/index.md [hunk 1]
```diff
@@ -3,22 +3,22 @@ layout: layout.pug
...
 
-Terraform is an open source infrastructure automation tool which uses templates to manage infrastructure for multiple public cloud providers, service providers, and on-premises solutions. Terraform creates your infrastructure, configures resources, and manages communication between agents. The purpose of this tool is to automate most of the manual efforts of managing and maintaining distributed systems.
+Terraform is an open source infrastructure automation tool which uses templates to manage infrastructure for multiple public cloud providers, service providers, and on-premises solutions. Terraform creates your infrastructure, configures resources, and manages communication between agents. The purpose of this tool is to automate most of the manual efforts of managing and maintaining distributed systems. The Universal Installer is built on top of Terraform.
 
-The primary goals of using Terraform are as follows:
+The primary goals of using Universal Installers are as follows:
```
Comment (yishan-lin) [inline pages/1.12/installing/evaluation/index.md]: The primary goals of using "The Universal Installer" (there is only one, which is our tool)

File: pages/1.12/installing/evaluation/index.md [hunk 2]
```diff
@@ -3,22 +3,22 @@ layout: layout.pug
 navigationTitle:  Evaluation
 title: Evaluation
 menuWeight: 10
-excerpt: Installing DC/OS for evaluation using Terraform templates
+excerpt: Installing DC/OS for evaluation using Universal Installer
 ---
 
 This page explains about Terraform and the evaluation methods that are supported for both DC/OS OSS (default) and DC/OS Enterprise installation. You can evaluate the installation of DC/OS cluster based on your requirements.
```
Comment (yishan-lin) [inline pages/1.12/installing/evaluation/index.md]: "This page explains about the Universal Installer..."

File: pages/1.12/installing/evaluation/index.md [hunk 3]
```diff
@@ -3,22 +3,22 @@ layout: layout.pug
...
 
-The primary goals of using Terraform are as follows:
+The primary goals of using Universal Installers are as follows:
 - Provide a single unified experience for provisioning, deploying, installing, upgrading, and decommissioning DC/OS on a cluster of machines. 
 - Create a modular and reusable script to easily decouple DC/OS on various OS and cloud providers to easily install, upgrade, and modify in-place..
 - Remove the confusion around which DC/OS installation method should be used in any given scenario. This automated tool helps to build modules that codify best practices for each stage in the cluster life-cycle and hook necessary modules into an existing infrastructure. 
 	
-Terraform is the recommended method for provisioning, deploying, installing, and upgrading DC/OS on a subset of our supported platforms.
+Universal Installer is the recommended tool for provisioning, deploying, installing, and upgrading DC/OS on a subset of our supported platforms.
```
Comment (yishan-lin) [inline pages/1.12/installing/evaluation/index.md]: The Universal Installer is the recommended tool...

File: pages/1.12/installing/index.md [hunk 4]
```diff
@@ -11,9 +11,11 @@ The installation of DC/OS involves configuring your infrastructure, and installi
 
 The DC/OS installation methods are as follows:
 
-- **Mesosphere supported installation:** These methods are used for trials and proofs of concept (PoC) only. To test or demo DC/OS on Azure, AWS, or GCP, follow the [Mesosphere supported installation methods](#mesosphere-supported).
+- **Mesosphere-supported installation:** Use the [Mesosphere-supported installation methods](#mesosphere-supported) to test or demo DC/OS on AWS, Azure, or GCP using an Universal Installer.
```
Comment (yishan-lin) [inline pages/1.12/installing/index.md]: Shouldn't this be "Mesosphere Supported Installation"?  no hyphens, capitalize

File: pages/1.12/installing/index.md [hunk 5]
```diff
@@ -11,9 +11,11 @@ The installation of DC/OS involves configuring your infrastructure, and installi
...
 
-- **Community supported installation:** These methods are used for trials and proofs of concept (PoC) only. To test or demo DC/OS on Digital Ocean or Packet, follow the [Community supported installation methods](#community-supported).
+- **Community-supported installation:** Use the [Community-supported installation methods](#community-supported) to test or demo DC/OS on AWS, Azure, Digital Ocean or Packet.
```
Comment (yishan-lin) [inline pages/1.12/installing/index.md]: Same things here, "Community Supported Installation"

File: pages/1.12/installing/index.md (hunk 4 above)
Comment (anithamuthu79) [reply pages/1.12/installing/index.md]: This is “Mesosphere-supported” and “Community-supported” (as they are compound adjectives).

**Pull request merged**